style.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:ital@0;1&display=swap');
+
+
+body
+{  
+    font-family: 'Playfair Display', serif;
+    background-color: #EDE9FE;;
+}
+
+.header {
+    grid-area: header;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 80vh;
+    color: black;
+    font-weight: bold;
+    border: 5px solid #A78BFA;
+    border-radius: 2rem;
+}
+
+.home-btn
+{
+    padding: 1rem;
+    border-radius: 1rem;
+    border: 2px solid black;
+    background-color: #C4B5FD;
+    cursor: pointer;
+}
+
+.home-btn:hover
+{
+    background-color: #DDD6FE;
```

Comment:
Hi Surbhi,
You can use color variables here.

Reply:
Done